skorch/tests/test_net.py

```diff
@@ -705,13 +705,13 @@ def test_target_1d_raises(self, net, data):
             "The target data shouldn't be 1-dimensional; "
             "please reshape (e.g. y.reshape(-1, 1).")
 
-    def test_predict_predict_proba(self, net, data):
+    def test_predict_predict_proba(self, net_fit, data):
         X = data[0]
-        y_pred = net.predict(X)
+        y_pred = net_fit.predict(X)
 
         # predictions should not be all zeros
         assert not np.allclose(y_pred, 0)
 
-        y_proba = net.predict_proba(X)
+        y_proba = net_fit.predict_proba(X)
```

Comment:
Why this change and why didn't the previous version fail? This test doesn't do much if the unfitted net behaves the same?

Reply:
The `net` fixture is uninitialized, which is why `predict` shouldn't work. Through side effects of other tests, this still works, but calling this test in isolation fails.

Reply:
And these side-effects are not a problem?